Library/Formula/llvm.rb

```diff
@@ -78,17 +92,25 @@ class Llvm < Formula
       url "http://llvm.org/git/lldb.git"
     end
 
-    resource "clang-tools-extra" do
-      url "http://llvm.org/git/clang-tools-extra.git"
+    resource "polly" do
+      url "http://llvm.org/git/polly.git"
+    end
+
+    resource "sanitizers" do
+      url "http://llvm.org/git/compiler-rt.git"
     end
   end
 
   option :universal
-  option "with-clang", "Build Clang support library"
+  option "with-clang", "Build the Clang compiler and support libraries"
+  option "with-extra-tools", "Build extra tools for Clang"
+  option "with-libcxx", "Build the libc++ standard library"
```

Comment:
The clang [get started](http://clang.llvm.org/get_started.html) page says that libcxx is "only required to build and run Compiler-RT tests on OS X, optional otherwise".

Then again, the documentation between the various projects is a bit shaky, but if the clang guys are fine building without it I think we're good